### ISSUE TITLE
PDF Not Loading

### ISSUE THREAD

**Issue by goz83** [j26design/PDF-Embedder-Premium#23]
I purchased the premium PDF Embedder and downloaded it. I had my Adobe Muse project open already and I double clicked the PDF Embedder file which then appeared in my Muse library. I dragged the widget to the page I wanted a PDF and I uploaded a PDF into the widget. 

I also uploaded the  web and build files (using file zilla) into the main web folder, the public_html and the website folder itself.

I then selected to view the site in browser and I get a "Page Not Found" message where the PDF image should display. This happens also when I select to preview the page in browser and has happened in Firefox, IE and Chrome. I have not checked others. 

This is the link address > http://127.0.0.1:33334/Preview/embassy-grill/menu.html

**Comment by jp26jp** [comment]
That's not a live website. Publish it to a live site and it will work.

**Comment by goz83** [comment]
Hi. I have uploaded the website to http://embassygrill.businesscatalyst.com/ which is a "real" website. It is still not working.

**Comment by goz83** [comment]
The PDF file is embedded under the menu images. Ideally the menu images will be deleted and replaced with a working PDF menu. Also the zoom function is set at 50 and I can't change it??

**Comment by jp26jp** [comment]
Publish the PDF embedder to an empty page

**Comment by goz83** [comment]
I have done this, Still same issue. See http://embassygrill.businesscatalyst.com/eat.html

I created a brand new page with nothing on it...not even a nav menu

**Comment by jp26jp** [comment]
If you send the login credentials to the server to john@j-26.com, I can take a look to see what's going on.

**Comment by goz83** [comment]
Login details sent a couple of days ago as requested.

**Comment by goz83** [comment]
Hi. Is there any update on this? I am hoping to get the website signed off before Christmas.

**Comment by jp26jp** [comment]
Where is a live version of the page? Login credentials you sent me were for GoDaddy but the page you said the widget is hosted is a business catalyst URL.

**Comment by goz83** [comment]
Hi.


I use GoDaddy Hosting for my regular websites. The catalyst site is the test site. To remove any confusion, I have:


1) Added a PDF to my main website at http://simplybuiltwebsites.ie/pdf-test.html


2) Added the web and build folders to the folder containing the website (simplybuiltwebsites)


3) Published the website again.


I am still seeing the Not Found error.


My websites are not in the public_html folder...out of habit, but this shouldn't have an effect on the file, as the Build and Web folders are also in the main folder containing all the websites.


The login details I provided should do the trick. I have also set up an ftp access for you if that's easier.

Host: 107.180.3.168

User: j25design@hostsbw.com

Pass: J26Design#


The website I really want it to work for is in folder embassygrill but I can't officially launch the site until I have it completed and know the PDF will work with it.


I hope this gives you the required info.


Gary


________________________________
